### PR TITLE
bt-464: Create cv and contacts component in admin user verficiation

### LINKED ISSUE #464: Create CV and Contacts component in admin user verification
Right now we don't have design so you can put some placeholders there

**User story:**
As an admin, I want to have a CV and Contacts tab on the Verification screen, so that I can see the candidate’s contact information

**Acceptance criteria:**

- There should be displayed all information which talent fills in on the onboarding step 03
- There should be displayed profile photo of the candidate, full name, phone number, Linkedin link and CV
- Admin should be able to go to the LinkedIn profile by clicking on the link
- Admin Should be able to open the CVs of candidates

### PR DESCRIPTION
**Pull request by JosuerBague**
Worked on making it responsive.

Plugin used: [react-doc-viewer](https://github.com/cyntler/react-doc-viewer)

Issue: TS error about DocViewer not having call signatures. But plugin runs with no problem on browser.
Slack Thread: https://bsa2023.slack.com/archives/C05NY1E7CBS/p1695218924964219

Desktop View:
image

Less than 1500px:
image

Less than 1300px:
image

Less than 900px:
image

Less than 500px (Mobile):
image

### CODE REVIEW

**Comment by JosuerBague** [comment]
This branch contains changes from #495

**Comment by nikita-remeslov** [comment]
Conflicts, Check design for sidebar

**Comment by JosuerBague** [comment]
> Conflicts, Check design for sidebar

Hello, I have sidebar setup to check if the user accessing the page is an admin. If user is an admin, they will have different sidebar.

image

image

image

File: frontend/src/bundles/admin-panel/admin-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import { PreviewTabs } from './constants/constants.js';
+import { employers, talents } from './mock-data/mock-data.js';
+import styles from './styles.module.scss';
+import { type FilterValues, type ValueOf } from './types/types.js';
+
+const AdminPanel: React.FC = () => {
+    const [filter, setFilter] = useState<FilterValues>('talents');
+    type TabValues = ValueOf<typeof PreviewTabs>;
+    const items = filter === 'talents' ? talents : employers;
+    const FIRST_INDEX = 0;
```
Comment (nika-sirak) [inline frontend/src/bundles/admin-panel/admin-panel.tsx]: move to constants file

File: frontend/src/bundles/admin-panel/admin-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import { VerificationList } from './components/components.js';
+import { CVAndContacts } from './components/cv-and-contacts/cv-and-contacts.js';
+import { PreviewTabs } from './constants/constants.js';
+import { employers, talents } from './mock-data/mock-data.js';
+import styles from './styles.module.scss';
+import { type FilterValues, type ValueOf } from './types/types.js';
+
+const AdminPanel: React.FC = () => {
+    const [filter, setFilter] = useState<FilterValues>('talents');
+    type TabValues = ValueOf<typeof PreviewTabs>;
```
Comment (nika-sirak) [inline frontend/src/bundles/admin-panel/admin-panel.tsx]: move to types folder

File: frontend/src/bundles/admin-panel/constants/preview-tabs.constant.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const PreviewTabs = {
```
Comment (nika-sirak) [inline frontend/src/bundles/admin-panel/constants/preview-tabs.constant.ts]: B23 https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

**Review by nika-sirak — COMMENTED** [review]
Did you add info from STEP 3? 
https://github.com/orgs/BinaryStudioAcademy/projects/17/views/1?pane=issue&itemId=38716403
image

**Review by nikita-remeslov — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**